src/store-tree/schema-placeholder.ls

```diff
@@ -48,10 +49,23 @@ is-map = ->
 
 get-type = ->
   | not is-placeholder it    => throw new Error "#{it} is not a placeholder"
-  | it is create-placeholder => SchemaType.ANY
   | otherwise                => it.type
 
 
+normalize = (input) ->
+  if input is create-placeholder
+    input!
+  else if input instanceof SchemaPlaceholder
+    input
+  else if input instanceof SchemaMap
+    input.child-schema = normalize input.child-schema
+    input
+  else if typeof! input is 'Object'
+    input |> Obj.map normalize
+  else
+    input
```

Comment:
This is to let it invalid schema's through the normalization process. It is currently caught later in the build `create-store` function. I'm fine throwing an error here as well